Formula/drafter.rb

```diff
@@ -0,0 +1,24 @@
+class Drafter < Formula
+  desc "Snow Crash parser harness"
+  homepage "https://apiblueprint.org/"
+  url "https://github.com/apiaryio/drafter/releases/download/v3.2.7/drafter-v3.2.7.tar.gz"
+  sha256 "a2b7061e2524804f153ac2e80f6367ae65dfcd367f4ee406eddecc6303f7f7ef"
+
+  def install
+    system "./configure", "--shared"
+    system "make", "install", "DESTDIR=#{prefix}"
+    bin.install "bin/drafter"
```

Comment:
This line is redundant because it's already been installed by the `make install` above.

Reply:
ping